docs/quickstart.rst

```diff
@@ -228,6 +228,11 @@ For automated backups the passphrase can be specified using the
     the key in case it gets corrupted or lost. Also keep your passphrase
     at a safe place.
 
+    You can make backups using :ref:`borg_key_export` subcommand.
+
+    If you want to print a backup of your key to paper use the ``--paper``
+    option of this command and print the result.
+
     The backup that is encrypted with that key/passphrase won't help you
     with that, of course.
```

Comment:
The "with that" here referred to "make a backup copy ... keep it at a safe place" some lines above.

Due to the new content you added in between, the reference is now somehow broken.